tcms/rpc/api/forms/testrun.py

```diff
@@ -36,39 +36,29 @@ def populate(self, product_id):
         self.fields['build'].queryset = Build.objects.filter(product_id=product_id, is_active=True)
 
 
-class NewExecutionForm(BaseCaseRunForm):
-    assignee = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
-    run = forms.ModelChoiceField(queryset=TestRun.objects.all())
-    case = forms.ModelChoiceField(queryset=TestCase.objects.all())
-
-    def clean_assignee(self):
-        data = self.cleaned_data.get('assignee')
-        if not data:
-            if self.cleaned_data.get('case') \
-                    and self.cleaned_data['case'].default_tester_id:
-                data = self.cleaned_data['case'].default_tester
-            elif self.cleaned_data.get('run') \
-                    and self.cleaned_data['run'].default_tester_id:
-                data = self.cleaned_data['run'].default_tester
+class NewTestExecutionForm(forms.ModelForm):
+    class Meta:
+        model = TestExecution
+        exclude = ('assignee', 'tested_by', 'status')
 
-        return data
+    assignee = UserField(required=False)
+    tested_by = UserField(required=False)
+    status = forms.ModelChoiceField(queryset=TestExecutionStatus.objects.all(), required=False)
```

Comment:
Drop this. The `status` field is required in the model and if the user doesn't pass it via the API the underlying model creation will break.